boards/adi/max32657evkit/Kconfig.max32657evkit

```diff
@@ -2,4 +2,5 @@
 # SPDX-License-Identifier: Apache-2.0
 
 config BOARD_MAX32657EVKIT
-	select SOC_MAX32657 if BOARD_MAX32657EVKIT_MAX32657
+	select SOC_MAX32657 if BOARD_MAX32657EVKIT_MAX32657 ||	\
+						   BOARD_MAX32657EVKIT_MAX32657_NS
```

Comment:
fix alignment

Reply:
My local tab size was 4, so missed this one, updated it as 8 and fixed. thanks.